### ISSUE TITLE
0.11 has wrong __version__ 

### ISSUE THREAD

**Issue by mattwelborn** [conda-forge/pint-feedstock#32]
Note: this is the same as hgrecco/pint#1038.

Pint v0.11 has the version string "0.0.0".

**Steps to reproduce:**
```
conda create -n pint11 -c conda-forge "pint==0.11"
conda activate pint11
python -c "import pint; print(pint.__version__)"
```
Output: `0.0.0`

**Additional context:**
Other versions have the correct version string.
```
conda create -n pint10 -c conda-forge "pint==0.10"
conda activate pint10
python -c "import pint; print(pint.__version__)"
```
Output: `0.10`

```
conda create -n pinthead pip python
conda activate pinthead
pip install git+https://github.com/hgrecco/pint
python -c "import pint; print(pint.__version__)"
```
Output: `0.12.dev8+g3aba60e`

**Comment by jan-janssen** [comment]
I have the same issue here https://github.com/conda-forge/matminer-feedstock/pull/3

**Comment by jan-janssen** [comment]
Unfortunately pip check does not check if the version of the package is consistent : https://github.com/conda-forge/pint-feedstock/pull/33

**Comment by dgasmith** [comment]
@conda-forge/python Any suggestions here with using Setuptools SCM and conda?

Only link I could find:
- https://github.com/pypa/setuptools_scm/issues/280

**Comment by scopatz** [comment]
There is an environment variable you can use to set the version number on the command line

**Comment by dgasmith** [comment]
@scopatz Thanks for the response! Do you have an example by chance? Trying to fix this in #34, but without much luck.
